tools/cldr-code/src/main/java/org/unicode/cldr/tool/GeneratePersonNameTestData.java

```diff
@@ -168,24 +167,28 @@ public static void main(String[] args) {
                     Multimap<String, String> valueToSource =
                             TreeMultimap.create(LENGTH_FIRST, Comparator.naturalOrder());
                     for (FormatParameters parameters : FormatParameters.allCldr()) {
-                        Options options;
                         Order order = parameters.getOrder();
+                        //                        Options options; // use this to get the 'none'
+                        // case
+                        //                        if (order == nameOrder) {
+                        //                            options = Options.none;
+                        //                        } else if (order == Order.sorting && nameOrder ==
+                        // myOrder) {
+                        //                            options = Options.sorting;
+                        //                        } else {
+                        //                            continue; // skip otherwise
+                        //                        }
 
-                        if (order == nameOrder) {
-                            options = Options.none;
-                        } else if (order == Order.sorting && nameOrder == myOrder) {
-                            options = Options.sorting;
-                        } else {
-                            continue; // skip otherwise
-                        }
-
-                        String formatted = formatter.format(entry.getValue(), parameters);
+                        String formatted =
+                                formatter
+                                        .format(entry.getValue(), parameters)
+                                        .replace("ᵛ", ""); // remove special CLDR ST hack
```

Comment:
Just for my own curiosity, what was the "CLDR ST hack"?  What did this character mean in that context?

Reply:
In CLDR, we don't have a way to substitute the vocative form of sample names, so there is a hack using the ᵛ to indicate that it would (if the implementation supports it) need the vocative form.

Reply:
Ah!  Okay.  Thank you!